### ISSUE TITLE
How to use the eatract_feature2.py to  calculate the similarity between two faces??

### ISSUE THREAD

**Issue by xiaosuzhang** [ZhaoJ9014/face.evoLVe.PyTorch#32]

**Comment by ZhaoJ9014** [comment]
Hi, you can use both Euclidean distance or cosine metric to compute the similarity score. For details, you can refer to the evaluation part in our codes. The performance of our released model has been reported in the table, please have a check. If your performance is not ideal enough, you can try to fine-tune the model on your own data or try some metric learning methods as a post-processing. Hope this helps. Thx.
